src/mpc/mpc_create_keypair.cpp

```diff
@@ -130,12 +131,13 @@ class mpc_create_keypair : public subcommand
         libff::leave_block("Load powers of tau");
 
         libff::enter_block("Load phase2 data");
-        if (!libff::inhibit_profiling_info) {
-            libff::print_indent();
-            std::cout << phase2_file << "\n";
-        }
-        srs_mpc_layer_C2<ppT> phase2 =
-            read_from_file<srs_mpc_layer_C2<ppT>>(phase2_file);
+        libff::print_indent();
+        std::cout << phase2_file << std::endl;
```

Comment:
I'd rename `phase2_file` as `phase2_challenge_file` since this file really is the challenge, right?